README.md

```diff
@@ -1,72 +1,56 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Devlopers Dev Summit 2022.
 
-> Description the project.
+> This is the capstone project from the first module of Microverse Program.
 
+This is a fictional page built for my portfolio project
+and is build based on an online website for a conference and Any Logos
+and name used is for the purpose of Placeholding. Website layout and design belongs to
+[Cindy Shin](https://www.behance.net/adagio07),and used for educational purposes.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
+- JavaScript
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+Here is how it looks [live](https://ishpaul777.github.io/Landing-page-for-Dev-summit/).
 
+## Loom Video
+I have presented the project in a [Loom Video](https://www.loom.com/share/89e7cef7a9cc4eaaaf2be69a8523f9c8).
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+### Get the repository into your PC.
 
-### Setup
-
-### Install
+- Clone this repository using the SSH key or download it as a ZIP from the "Code" tab and extract the files.
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
+- To use it open the "index.html" file in your local copy of the repository after cloning or downloading.
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Ishpaul Singh**
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- [@github](https://github.com/ishpaul777)
+- [@twitter](https://twitter.com/ishpaul777)
+- [@Linkedin](https://www.linkedin.com/in/ishpaul-singh-264590226/)
```

Comment:
While you did address the previous reviewer's required change, you didn't do so properly. Make sure the content that you are linking (that means the content inside the []) includes your name or your social media handle. Having these links with the social media name is not professional and it doesn't give you visibility.